src/insexpand.c

```diff
@@ -4412,6 +4464,51 @@ get_next_default_completion(ins_compl_next_state_T *st, pos_T *start_pos)
     return found_new_match;
 }
 
+/*
+ * Return the callback function associated with "funcname".
+ */
+#ifdef FEAT_COMPL_FUNC
+    static callback_T *
+get_cpt_func_callback(char_u *funcname)
+{
+    static callback_T	cb;
+    char_u		buf[LSIZE];
+    int			slen;
+    static char_u	*name;
```

Comment:
```suggestion
    static char_u	*name = NULL;
```

And then below always use `vim_free(name)` ?

Reply:
> And then below always use vim_free(name) ?

There is some bug in free_callback(). Before, I had `name` as non-static, and it was getting freed inside `free_callback()` because `cb_free_name` was set. But it resulted in memory leak according one of your CI checks. So, I changed to this strategy to pass the check. I will try to make it non-static again and test it out.